### PR TITLE
feature/UI

### PR DESCRIPTION
**Pull request by ngocphong230801**
In this pull request:
- Create UI for web app

### CODE REVIEW

File: practice/src/index.html [hunk 1]
```diff
@@ -7,10 +7,58 @@
...
+                    Build from <a href="https://www.typescriptlang.org/" class="underline-text" target="_blank">TypeScript</a>
+                    and <a href="https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller" class="underline-text" target="_blank">MVC model</a>
+                </p>
+            </header>
+            <div class="app-todo-content">
+                <div class="content-input">
+                    <i class="fa fa-check check-all"></i>
+                    <input id="task-input" class="task-input" type="text" placeholder="What will you focus on today?">
+                </div>
+                <ul class="list-task"></ul>
```
Comment (lintran-agilityio) [inline practice/src/index.html]: what is purpose this code?

File: practice/src/index.html [hunk 2]
```diff
@@ -7,10 +7,58 @@
...
+            <header class="app-heading">
+                <h1 class="heading-title">Todo list with TypeScript</h1>
+                <p class="heading-description">
+                    Build from <a href="https://www.typescriptlang.org/" class="underline-text" target="_blank">TypeScript</a>
+                    and <a href="https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller" class="underline-text" target="_blank">MVC model</a>
+                </p>
+            </header>
+            <div class="app-todo-content">
+                <div class="content-input">
+                    <i class="fa fa-check check-all"></i>
```
Comment (lintran-agilityio) [inline practice/src/index.html:22]: should close tag if it isn't have content after
example `<i class="fa fa-check check-all" />`
the same another case

File: practice/src/index.html [hunk 3]
```diff
@@ -7,10 +7,58 @@
...
+                </div>
+                <ul class="list-task"></ul>
+                <div class="content-action">
+                    <p class="total-item"></p>
+                    <ul class="task-filter">
+                        <li class="task-filter-item"><a data-action="all" href="#all" class="selected item all active-btn" tabindex="0">All</a></li>
+                        <li class="task-filter-item"><a data-action="active" href="#active" class="active item selected" tabindex="0">Active</a></li>
+                        <li class="task-filter-item"><a data-action="completed" href="#completed" class="completed item selected" tabindex="0">Completed</a></li>
+                    </ul>
+                    <a href=""><p class="clear-completed">Clear completed</p></a>
```
Comment (lintran-agilityio) [inline practice/src/index.html]: the inline tag should n't wrapper block tag

File: practice/src/index.html (hunk 1 above)
Comment (ngocphong230801) [reply practice/src/index.html]: When I perform the function of adding a new task
then the task list will be added in this tag list.

File: practice/src/index.html [hunk 4]
```diff
@@ -7,10 +7,60 @@
...
+                    <!-- Todo list task item will be added here  -->
+                </ul>
+                <div class="content-action">
+                    <p class="total-item"></p>
+                    <ul class="task-filter">
+                        <li class="task-filter-item"><a data-action="all" href="#all" class="selected item all active-btn" tabindex="0">All</a></li>
+                        <li class="task-filter-item"><a data-action="active" href="#active" class="active item selected" tabindex="0">Active</a></li>
+                        <li class="task-filter-item"><a data-action="completed" href="#completed" class="completed item selected" tabindex="0">Completed</a></li>
+                    </ul>
+                    <p class="clear-completed">Clear completed</p>
```
Comment (lintran-agilityio) [inline practice/src/index.html]: do you have handle clear completed feature?

Comment (ngocphong230801) [reply practice/src/index.html]: Yes, It is a feature of the app

File: practice/src/styles/components/confirm-delete.css [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    margin-top: -10px;
+    width: 290px;
+    height: 32px;
+    font: inherit;
+    vertical-align: baseline;
+}
+
+.confirm-dialog .confirm-action {
+    display: flex;
+    border-top: 0.1px solid #bfc5bf;
```
Comment (lintran-agilityio) [inline practice/src/styles/components/confirm-delete.css]: should define color variables to reuse and update

Comment (lintran-agilityio) [reply practice/src/styles/components/confirm-delete.css]: same another colors

File: practice/src/styles/components/loading.css [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+    -webkit-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    transform: rotate(45deg);
+    -webkit-animation: spin 1.5s infinite;
+    animation: spin 1.5s infinite;
+}
+
+.preloader__processing {
+    position: absolute;
+    top: 65%;
```
Comment (lintran-agilityio) [inline practice/src/styles/components/loading.css:42]: why do you know the position absolute top is 65%?

File: practice/src/styles/components/loading.css [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+    margin-left: -8px;
```
Comment (lintran-agilityio) [inline practice/src/styles/components/loading.css:43]: because `preloader__processing` is absolute so you should use `left` intead.

**Review by lintran-agilityio — APPROVED** [review]

**Pull request merged**